### DISCUSSION TITLE
GLDAS forcing data. Without February 29

### DISCUSSION THREAD

**Discussion by Zhoujieliang** [NCAR/noahmp#150] [General]
After ' perl create_UV_netcdf.perl ', the file without February 29th.
![image](https://github.com/user-attachments/assets/23dc5122-ee34-41ab-88c3-4c932772db8b)

**BUT,** for ' python extract_gldas.py', there are February 29th.
![image](https://github.com/user-attachments/assets/a606c3d3-771f-4ce0-af56-7f5e1ddd67bc)

**Comment by Zhoujieliang** [comment]
What causes this and does it have an effect on the production drive data

**Reply by cenlinhe** [reply]
This might be a bug for create_UV_netcdf.perl. @CharlesZheZhang Have you seen this issue before?
